### PR TITLE
Multiple client instances integration tests

### PR DESCRIPTION
**Pull request by slvrtrn**
#55 

Common
--
- [x] Can create several instances on a language client
- [x] Clients can send parallel requests (DDL, select, insert stream, insert array)

### CODE REVIEW

File: __tests__/integration/multiple_clients.test.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      clients.map((client, i) =>
+        client
+          .select({
+            query: `SELECT * FROM system.numbers LIMIT ${i + 1}`,
+            format: 'JSONEachRow',
+          })
+          .then(async (r) => {
+            results[i] = (await r.json<Res>())
+              .map((r) => +r.number)
+              .reduce((prev, cur) => prev + cur)
```
Comment (mshustov) [inline __tests__/integration/multiple_clients.test.ts:36]: Isn't asserting a result it received enough? What does `reduce` give us?

File: __tests__/integration/multiple_clients.test.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    }
+  })
+
+  afterEach(async () => {
+    for (const c of clients) {
+      await c.close()
+    }
+  })
+
+  it('should send multiple parallel selects', async () => {
```
Comment (mshustov) [inline __tests__/integration/multiple_clients.test.ts:23]: we can add a test that if a request is aborted with `signal`, it doesn't affect other parallel requests

File: __tests__/integration/multiple_clients.test.ts (hunk 1 above)
Comment (slvrtrn) [reply __tests__/integration/multiple_clients.test.ts:36]: just to have a single number as a result. It was handy when I tested it with 20 client instances causing result sets to become really lengthy.

File: __tests__/integration/multiple_clients.test.ts (hunk 2 above)
Comment (slvrtrn) [reply __tests__/integration/multiple_clients.test.ts:23]: I will add this test in the scope of the "single client - multiple requests" test

**Pull request merged**